frontend/src/reducers/user.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  login: {
+    accessToken: "",
+    userId: "",
+    alias: "",
+  },
+};
+
+export const user = createSlice({
+  name: "user",
+  initialState,
+  reducers: {
+    setAccessToken: (state, action) => {
+      const { accessToken } = action.payload;
+      state.login.accessToken = accessToken;
+    },
+    setUserId: (state, action) => {
+      const { userId } = action.payload;
+      state.login.userId = userId;
+    },
+    setAlias: (state, action) => {
+      const { alias } = action.payload;
+      state.login.alias = alias;
+    },
+    deleteAccessToken: (state, action) => {
+      state.login.accessToken = "";
+    },
+    deleteUserId: (state, action) => {
+      state.login.userId = "";
+    },
+  }
+});
```

Comment:
Reducer is a great place to manage your users but you need to store some data in the local storage to make them stay legged in after they refresh the page 🤓